client/packages/common/src/hooks/useUrlQuery/useUrlQueryParams.ts

```diff
@@ -76,7 +76,7 @@ export const useUrlQueryParams = ({
     offset: urlQuery.page ? (urlQuery.page - 1) * rowsPerPage : 0,
     first: rowsPerPage,
     sortBy: {
-      key: urlQuery.sort ?? initialSort,
+      key: urlQuery.sort ?? initialSort?.key ?? '',
```

Comment:
offroad: the  sort was being set to an object which then caused an error in the graphql query, which required re-querying. This tidies that up.